### PR TITLE
Session type selection

### PR DESCRIPTION
**Pull request by kajsaunge**
Issue: [Session type selection](https://www.notion.so/29k/Closed-BETA-0055aabd797b40c7b390b7cea4eba658#4c0b598b7cb64fb59a5042ac69079ca2)

### Description of the Change

- Session type icon
- New design / layout
- Show user profile

#### Screenshots

| iOS | Android |
|-|-|
| image | image |

### CODE REVIEW

**Review by Nipher — APPROVED** [review]
👍

File: client/src/routes/CreateSessionModal/CreateSessionModal.tsx [hunk 1]
```diff
@@ -295,6 +319,10 @@ const CreateSessionModal = () => {
   );
 
   const hasProfile = Boolean(user?.displayName) && Boolean(user?.photoURL);
+  const userProfile = {
+    displayName: user?.displayName ?? undefined,
+    photoURL: user?.photoURL ?? undefined,
```
Comment (gewfy) [inline client/src/routes/CreateSessionModal/CreateSessionModal.tsx:324]: Do you really need the `??` here?

**Review by gewfy — CHANGES_REQUESTED** [review]
Added som comments

File: client/src/routes/CreateSessionModal/CreateSessionModal.tsx [hunk 2]
```diff
@@ -77,8 +85,16 @@ const Step = styled(Animated.View).attrs({
...
-  alignItems: 'center',
+  justifyContent: 'center',
+  height: 96,
   flex: 1,
+  backgroundColor: COLORS.PURE_WHITE,
+  borderRadius: SPACINGS.SIXTEEN,
+  paddingHorizontal: SPACINGS.SIXTEEN,
+});
+const TypeItemHeading = styled(ModalHeading)({
+  alignSelf: 'flex-start',
```
[the remark this answers is not in the document]
Comment (kajsaunge) [reply client/src/routes/CreateSessionModal/CreateSessionModal.tsx]: Riiight on! 🙏

File: client/src/routes/CreateSessionModal/CreateSessionModal.tsx (hunk 1 above)
Comment (kajsaunge) [reply client/src/routes/CreateSessionModal/CreateSessionModal.tsx:324]: Yeah, else typeScript complains it can be null. Or is there another way @gewfy ?

**Review by gewfy — APPROVED** [review]

**Pull request merged**